packages/react-scripts/template/src/setupTests.js

```diff
@@ -4,11 +4,15 @@
 
 /* eslint-disable import/no-extraneous-dependencies */
 import { cleanup } from '@fs/zion-testing-library'
+import { act } from 'react-dom/test-utils'
 import '@testing-library/jest-dom/extend-expect'
 
 // This performs a global cleanup as recommended by Kent C. Dodds.
 // For more details see: https://bit.ly/2FhMiUT
-afterEach(cleanup)
+afterEach(() => async () => {
+  await act(async () => {})
+  cleanup()
+})
```

Comment:
https://facebook.github.io/create-react-app/docs/running-tests and react testing library recommends:
```
import 'react-testing-library/cleanup-after-each'
```
But since we wrap it we can't do that, and they didn't expose directly the internal that it uses, so I looked up their implementation and did my best. 

However, this may not be necessary any more:
https://github.com/testing-library/react-testing-library/releases/tag/v9.0.0